tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @author Simon Heybrock
+import pytest
+import scipp as sc
+
+
+def make_variable() -> sc.Variable:
+    v = sc.arange('dummy', 12, dtype='int64')
+    return v.fold(dim='dummy', sizes={'xx': 4, 'yy': 3})
+
+
+def make_array() -> sc.DataArray:
+    da = sc.DataArray(make_variable())
+    da.coords['xx'] = sc.arange('xx', 4, dtype='int64')
+    da.coords['yy'] = sc.arange('yy', 3, dtype='int64')
```

Comment:
Because we once had a bug in the plotting code that hard-coded 'x', so we had a bunch of tests that passed only because we used 'x' as dim label in the test data.